src/ontology/slurp/omim.tsv

```diff
@@ -2,25 +2,13 @@ mondo_id	mondo_label	xref	xref_source	original_label	definition	parents
 ID	LABEL	A oboInOwl:hasDbXref	>A oboInOwl:source SPLIT=|		A IAO:0000115	SC %
 MONDO:0958222	maple syrup urine disease,  iia 1b	OMIM:620698	MONDO:equivalentTo	maple syrup urine disease,  iia 1b		
 MONDO:0958223	maple syrup urine disease,  iia 2	OMIM:620699	MONDO:equivalentTo	maple syrup urine disease,  iia 2		
-MONDO:0958224	encephalopathy, porphyria-related	OMIM:620704	MONDO:equivalentTo	encephalopathy, porphyria-related		
-MONDO:0958225	epidermolytic hyperkeratosis 2b, autosomal recessive	OMIM:620707	MONDO:equivalentTo	epidermolytic hyperkeratosis 2b, autosomal recessive		
-MONDO:0958226	leukoencephalopathy, porphyria-related	OMIM:620711	MONDO:equivalentTo	leukoencephalopathy, porphyria-related		
+MONDO:0958225	epidermolytic hyperkeratosis 2b, autosomal recessive	OMIM:620707	MONDO:equivalentTo	epidermolytic hyperkeratosis 2b, autosomal recessive		MONDO:0957316
```

Comment:
## QC: `build-mondo-ingest`
I'm currently running the build and will open up a data PR and link it here when it is ready.

Reply:
Oh wow you already did this! Forget my comments in the OMIM repo then. Good job.

Reply:
Sounds good; gonna disregard [this comment](https://github.com/monarch-initiative/omim/issues/103#issuecomment-2026950154) then.. Glad this looks as you would expect.

This is approved, so I am merging.